pipeline/pipeline.yaml

```diff
@@ -3,16 +3,15 @@ Description: "Template for full CI/CD serverless applications."
 Parameters:
   AppName:
     Type: String
-    Description: Name of the application.
-  SAMInputFile:
-    Type: String
-    Description: The filename for the SAM file.
+    Default: lambda-file-refarch
+    Description: Name used for application deployment
   SAMOutputFile:
     Type: String
+    Default: packaged-template.yml
     Description: The filename for the output SAM file from the buildspec file
   CodeBuildImage:
     Type: String
-    Default: "aws/codebuild/nodejs:4.3.2"
+    Default: "aws/codebuild/amazonlinux2-x86_64-standard:1.0"
     Description: Image used for CodeBuild project.
   GitHubRepoName:
```

Comment:
Adding in a default value of `lambda-refarch-fileprocessing` or removing this as a parameter would help simplify the deployment.

Reply:
Yes of course, the repo will be public so ultimately we could add defaults for the owner as well.